domain/machine/state/types.go

```diff
@@ -78,7 +78,7 @@ type instanceID struct {
 // data columns of status and status_data tables for both machine and machine
 // cloud instances within the sqlair statements in the machine domain.
 type machineStatusWithData struct {
-	Status  int        `db:"status"`
+	Status  int        `db:"status_id"`
```

Comment:
Should you update the field name on the struct as well so it is clear what the value is ?

Reply:
Yeah. Still still more fixes to here here for this stuff. This is a start.